src/holodeck/util.py

```diff
@@ -160,3 +161,26 @@ def draw_point(env, loc, color=None, thickness=10.0):
     color = [255, 0, 0] if color is None else color
     command_to_send = DebugDrawCommand(3, loc, [0, 0, 0], color, thickness)
     env._enqueue_command(command_to_send)
+
+
+def logs_path():
```

Comment:
Could you rename this to `log_paths`? I think that makes more sense. Once you get everything formatted, LGTM!